docs/features.md

```diff
@@ -280,13 +280,13 @@ environments:
         #   - not newsletterSubscribers
         segments:
           - and:
-            - iphoneUsers
-            - adultUsers
+              - iphoneUsers
+              - adultUsers
```

Comment:
can we verify if the extra added spaces will not result into unexpected behaviour in final parsed JSON?